### ISSUE TITLE
Case sensitive error

### ISSUE THREAD

**Issue by Renato66** [cweijan/vscode-mysql#764]
Could auto complete infers that this is an case sensitive and place some quotes?
it also happens for column names
Im using postgres as database

<img width="327" alt="image" src="https://user-images.githubusercontent.com/9284273/217106866-460470d9-78ee-42da-a9fb-866a9a98a4ed.png">

actual result:
<img width="305" alt="image" src="https://user-images.githubusercontent.com/9284273/217106492-d12fe547-f162-4aa7-893c-78096d06d36d.png">

expected:
<img width="278" alt="image" src="https://user-images.githubusercontent.com/9284273/217106547-fff49c1e-dbb6-4156-a2e8-aa678ce22d6d.png">

**Comment by cweijan** [comment]
Please check if you have modified this setting, it will be automatically wrapped by default.

![image](https://user-images.githubusercontent.com/27798227/217121095-01337776-446c-47da-8683-1872aa0a17d6.png)

**Comment by Renato66** [comment]
Did test with true/false/always/auto and nothing =/
some more details:

```
Version: 1.75.0 (Universal)
Commit: e2816fe719a4026ffa1ee0189dc89bdfdbafb164
Date: 2023-02-01T15:24:42.903Z (5 days ago)
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Darwin arm64 22.2.0
Sandboxed: No
```

```
MacOs 13.1 (22C65)
```

```
database-client v6.1.7 
```

**Comment by cweijan** [comment]
Sorry I didn't notice that you are completing on the result page, this option only takes effect in the VSCode editor, I will support it in future versions.

**Comment by cweijan** [comment]
Fixed in version 6.1.8.
